lib/src/in_app_purchase/in_app_purchase.dart

```diff
@@ -85,19 +102,20 @@ class InAppPurchaseController extends ChangeNotifier {
 
   /// Subscribes to the [inAppPurchaseInstance.purchaseStream].
   void subscribe() {
+    _subscription?.cancel();
     _subscription =
         inAppPurchaseInstance.purchaseStream.listen((purchaseDetailsList) {
       _listenToPurchaseUpdated(purchaseDetailsList);
     }, onDone: () {
       _subscription?.cancel();
-    }, onError: (error) {
+    }, onError: (dynamic error) {
```

Comment:
Why this change?

Reply:
this change should not be there, I'll revert